lib/helper/shtml.js

```diff
@@ -35,18 +25,21 @@ module.exports = function shtml(val) {
           return;
         }
 
-        value = String(value);
+        value = value.toString();
```

Comment:
use `String(value)` to avoid `value.toString` not exist

Reply:
Fixed